doc/source/reference/c-api.types-and-structures.rst

```diff
@@ -6,7 +6,7 @@ Python Types and C-Structures
 
 Several new types are defined in the C-code. Most of these are
 accessible from Python, but a few are not exposed due to their limited
-use. Every new Python type has an associated :c:type:`PyObject *` with an
+use. Every new Python type has an associated :c:type:`PyObject *<PyObject>` with an
```

Comment:
A generic `PyObject` is a C `struct` that begins with a `PyObject` (i.e., subclasses PyObject), which in turn is a struct that contains a reference count and a pointer to a `PyObjectType` struct. I don't know how that fits with the rest of the paragraph 8-)